.gitignore

```diff
@@ -6,3 +6,4 @@ dist
 .eslintcache
 *.log
 /package-lock.json
+.idea
```

Comment:
I still don't like this. None of our repos have this

Reply:
can take it out

Reply:
let's do that 🙏